.github/workflows/go.yml

```diff
@@ -2,7 +2,7 @@ name: ci
 
 on:
   push:
-    branches: '**'
```

Comment:
@directionless , this means now CI will only run in my forked repo when I push to a main or master branch ... do we loose anything by having it the way it was?

Reply:
It was causing two sets of tests to run on the PRs. One triggered for the push, and one triggered for the PR. 

For my branches, I usually push to a draft PR. But maybe there's a way we can get the best of both worlds. I wonder if we can specify fork names or something